pkg/controller/builder/pod_builder_test.go

```diff
@@ -67,7 +68,7 @@ func TestBuildActiveGatePodSpecs(t *testing.T) {
 
 		container := specs.Containers[0]
 		assert.Equal(t, ActivegateName, container.Name)
-		assert.Equal(t, ActivegateImage, container.Image)
+		assert.Equal(t, "/linux/activegate", container.Image)
```

Comment:
Add an apiUrl to the Spec object so that this has a proper domain. If there wasn't an apiUrl something should have broken long before (i.e., always expect apiUrl to be set for this function.)

Reply:
Adjust AG instance and added a proper check for the image name